packages/common/src/interceptors.ts

```diff
@@ -18,6 +18,7 @@ export type Headers = Record<string, Payload>;
  * @param method the name of the interceptor method to compose
  * @param next the original function to be executed at the end of the interception chain
  */
+// ts-prune-ignore-next (imported via lib/interceptors)
```

Comment:
Can you explain why this needs to be ignored?

Reply:
`ts-prune` doesn't see that this function is used. the compiled version is used (imported via `common/lib/interceptors`). I read the ts-prune readme, but didn't see how to tell it to include `lib/` imports in its search, or this relative import:

<img width="1058" alt="image" src="https://user-images.githubusercontent.com/251288/194668863-5e79c227-762d-41ef-9926-13292cdff454.png">